index.js

```diff
@@ -69,15 +69,21 @@ Limiter.prototype.get = function (fn) {
     .zcard([key])
     .zadd([key, now, now])
     .zrange([key, 0, 0])
+    .zrange([key, -max, -max])
     .pexpire([key, duration])
     .exec(function (err, res) {
       if (err) return fn(err);
-      var count = parseInt(Array.isArray(res[0]) ? res[1][1] : res[1]);
-      var oldest = parseInt(Array.isArray(res[0]) ? res[3][1] : res[3]);
+
+      var isIoRedis = Array.isArray(res[0]);
```

Comment:
oh, now I understood better why this check is necessary. I removed it from the promise equivalent package:

https://github.com/tj/node-ratelimiter/pull/44/files#diff-168726dbe96b3ce427e7fedce31bb0bcR77

Making it a bit faster, thanks 🙂

(I will add this PR as well)